ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -120,6 +120,48 @@ Item {
                        + bannerColumn.height
                        + Style.current.bigPadding
 
+        background: MouseArea {
+            acceptedButtons: Qt.RightButton
+            onClicked: {
+                popup.x = mouse.x
+                popup.y = mouse.y
+                popup.open()
+            }
+
+        property var popup: StatusPopupMenu {
+            StatusMenuItem {
+                //% "Create channel"
+                text: qsTrId("create-channel")
+                icon.name: "channel"
+                // Not Refactored Yet
+                enabled: communityData.amISectionAdmin
+                onTriggered: Global.openPopup(createChannelPopup)
+            }
+
+            StatusMenuItem {
+                //% "Create category"
+                text: qsTrId("create-category")
+                icon.name: "channel-category"
+                enabled: communityData.amISectionAdmin
+                onTriggered: Global.openPopup(createCategoryPopup)
+            }
+
+            StatusMenuSeparator {}
+
+            StatusMenuItem {
+                //% "Invite people"
+                text: qsTrId("invite-people")
+                icon.name: "share-ios"
+                enabled: communityData.canManageUsers
+                onTriggered: Global.openPopup(inviteFriendsToCommunityPopup, {
+                                                  community: communityData,
+                                                  hasAddedContacts: root.hasAddedContacts,
+                                                  communitySectionModule: root.communitySectionModule
+                                              })
+            }
```

Comment:
Just for you to know:

`StatusChatListAndCategories` already has a slot for a popup menu that is opened on right click here: https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Components/StatusChatListAndCategories.qml#L55-L60

This is actually one of the (few) reasons `StatusChatListAndCategories` exists. You can see how it can be used here: https://github.com/status-im/StatusQ/blob/master/sandbox/demoapp/StatusAppCommunityView.qml#L144-L161

Reply:
I understand, but its about out-of-component clicks